packages/cli/test/functional/test_site/expected/index.html

```diff
@@ -608,9 +600,7 @@ <h2 id="panel-with-src-from-another-markbind-site-header-2"><span id="panel-with
             </template></panel>
         </div>
         <p><strong>Modal with panel inside</strong></p>
-        <p>
-          <trigger for="modal-with-panel">trigger</trigger>
-        </p>
+        <trigger for="modal-with-panel">trigger</trigger>
```

Comment:
missed this previously: let's add in popover / trigger / tooltip to the inline tags list as well